script.js

```diff
@@ -24,3 +19,13 @@ function topFunction() {
   document.body.scrollTop = 0; // For Safari
   document.documentElement.scrollTop = 0; // For Chrome, Firefox, IE and Opera
 }
+
+
+// Open Close nav 
+function openNav() {
+  document.getElementById("mySidenav").style.width = "250px";
```

Comment:
You can also use a CSS class here if you prefer - that'll allow you to do some nice animations using CSS transitions. Totally up to you though, this is valid code.